GOVERNANCE-subprojects.md

```diff
@@ -18,6 +18,48 @@ The [TODO:PROJECTNAME] umbrella project is composed as a federation of individua
 some independent and some interdependent, each of which focuses on some aspect
 of [TODO:Goals of umbrella project].  Our governance reflects this federated structure.
 
+- [Values](#values)
+- [Individual Project Governance](#individual-project-governance)
+- [Steering Committee](#steering-committee)
+  - [Steering Committee Duties](#steering-committee-duties)
+  - [Steering Committee Elections](#steering-committee-elections)
+- [Code of Conduct Committee](#code-of-conduct-committee)
+- [Adding New Subprojects](#adding-new-subprojects)
+- [Removing Projects](#removing-projects)
+
+## Values
+
+<!-- This is where you put the core values or principles of your project, like
+openness, distributed design, fairness, diversity, etc.
+
+References and Examples
+* https://www.apache.org/theapacheway/
+* https://kubernetes.io/community/values/
+
+See https://contribute.cncf.io/maintainers/governance/charter for guidance and
+additional examples.  The values below are just example values as a jumping-off
+point for your project's actual values.  -->
+
+The [TODO: PROJECTNAME] and its leadership embrace the following values:
+
+* Openness: Communication happens in the open and is archived for future
```

Comment:
is there a better word for `archived`? we meed `discoverable` or `find-able` or `search-able` right? (archive could mean backed up?)

Reply:
Excellent point. I'll replace it with "discoverable" in all 3 templates.